anyway/infographics_utils.py

```diff
@@ -1565,8 +1688,11 @@ def get_latest_accident_date(table_obj, filters):
 def generate_widgets(request_params: RequestParams, to_cache: bool = True) -> List[Widget]:
     widgets = []
     # noinspection PyArgumentList
+    is_urban = is_location_inner_city(request_params.location_info)
     for w in get_widget_factories():
         widget: Widget = w(request_params)
+        if is_urban != widget.is_urban():
+            continue
```

Comment:
Urban is the first categorization of a newsflash by which a widget decides whether it is relevant. Soon we will have many additional categories. The different categories should not appear on the infrastructure level. I have two suggestions until the ranking will be operational:
1 - Simply return empty items, and `generate_widgets()` will not enter it to the results
2 - instead of `is_urban(location_info)`, use `is_included(request_params)`